spec/account/account-service.spec.js

```diff
@@ -1,32 +1,44 @@
 "use strict";
 
 describe("AccountService", () => {
-    var accountService, storageFake, promiseMock, secureHash;
+    var accountService, promiseMock, secureHash, StorageFake;
 
     beforeEach(() => {
-        var AccountService, config;
+        var config;
 
-        AccountService = require("../../lib/account/account-service");
         promiseMock = require("../mock/promise-mock");
-        storageFake = jasmine.createSpyObj("storageFake", [
-            "configure",
-            "getAsync",
-            "putAsync"
-        ]);
-        storageFake.configure.andCallFake(() => {
-            return storageFake;
-        });
-        storageFake.getAsync.andCallFake(() => {
-            return promiseMock.resolve(
-                new Buffer('{"data": "thing"}', "binary")
-            );
-        });
-        storageFake.putAsync.andCallFake(() => {
-            return promiseMock.resolve(true);
-        });
+        StorageFake = class StorageFake {
```

Comment:
Remember to consider what we should do to make this an instance instead of a class.